### ISSUE TITLE
se tags to better organize metrics on the UI.

### ISSUE THREAD

**Issue by motime** [Recognos/Metrics.NET#81]
In the (hopefully near) future the visualization app will use tags to better organize metrics on the UI.

**Comment by PaulParau** [comment]
No, we do not have any concrete plans for that. Feel free to work on a pull request if you want to :)

FYI the repo for the visualization app is [here](https://github.com/Recognos/Metrics.NET.FlotVisualization).

**Comment by motime** [comment]
10x
